sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/metrics/FetchExecutionRangeAccumulator.java

```diff
@@ -11,7 +11,7 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Accumlator that acts as a builder of FetchExecutionRanges
+ * Accumlator that acts as a changeFeedProcessorBuilder of FetchExecutionRanges
```

Comment:
undo, seems you did a replace all on all builder string to changeFeedProcessorBuilder.

Please also search for similar incidents elsewhere.

Reply:
Corrected